code/modules/hallucinations/effects/blind_rush_hallucination.dm

```diff
@@ -0,0 +1,69 @@
+/**
+  * # Hallucination - Blind Rush
+  *
+  * Makes target blind and causes them to see semi-transparent humanoids running at them.
+  */
+/datum/hallucination_manager/blind_rush
+	initial_hallucination = /obj/effect/hallucination/no_delete/blind_rusher
```

Comment:
Fairly confident this does effectively nothing. `spawn_hallucination` overrides it with a new instance of blind_rush hallucination. This is likely meant to be the instance of hallucination itself, not specific effects of it, but I'm not 100% certain how this new system works.

Reply:
This is how you should use it, the initial hallucination uses this typepath to decide what hallucination to spawn in